src/components/Search/Search.js

```diff
@@ -36,7 +35,7 @@ const styles = StyleSheet.create({
     fontWeight: 'normal',
     fontSize: 13,
     letterSpacing: -0.28,
-    color: COLORS.silver,
+    color: '#BDBDBD',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider using a centralized color configuration instead of hardcoding the color value.
```diff
-    color: '#BDBDBD',
+    color: colors.silver,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->